app.js

```diff
@@ -1,67 +1,7 @@
+let hamburger = document.querySelector('.hamburger');
+let nav = document.querySelector('.menulist');
 
-
-
-//Gets all elements IDs and classes
-let hamburger = document.getElementById('hamburger');
-let menuList = document.getElementById('menulist');
-let section = document.querySelectorAll('.top, .fordesktop, .language');
-let items = document.querySelectorAll('.elements ul *')
-
-
-let styles = {
-    display: 'block',
-    position: 'absolute',
-    right: '-5px',
-    top: '-5px',
-    width: '100vw',
-    height: '100vh',
-    textAlign: 'center',
-    margin: '0',
-    paddingTop: '80px',
-
-    background: 'rgba(0, 47, 255, 0.54)',
-    boxShadow: '0 4px 30px rgba(0, 0, 0, 0.1)',
-    backdropFilter: 'blur(6.7px)',
-}
-
-let unstyle = {
-    background: 'none'
-}
-
-hamburger.addEventListener('click', function () {
-
-    if (menuList.style.display == 'none') {
-        Object.assign(menuList.style, styles);
-
-       items.forEach(function(each) {
-        each.style.marginTop = '40px';
-        each.style.textDecoration = 'none';
-        each.style.color = 'white';
-        each.style.fontWeight = '700';
-        each.style.fontSize = '2.5rem';
-       })
-
-
-    }
-    else {
-        menuList.style.display = 'none'
-    }
-})
-
-function clickOut() {
-    menuList.style.display = 'none'
-}
-
-
-menuList.addEventListener('click', clickOut);
-
-window.addEventListener('resize', function(){
-    if(window.innerWidth >= 768) {
-        menuList.style.display = 'flex'
-    }
-
-    else {
-        menuList.style.display = 'none'
-
-    }
-})
\ No newline at end of file
+hamburger.addEventListener('click', function(){
+    hamburger.classList.toggle('active');
+    nav.classList.toggle('active');
+});
```

Comment:
- [ ] Your menu opens and closes fine and has a nice design, but as the project required, after the user clicks on the menu links, he/she should be redirected to the correct part of the page and the menu should close. Currently, the menu is not closing. Please implement this functionality to your project.